packages/contracts-bedrock/src/L2/L1BlockInterop.sol

```diff
@@ -104,28 +93,18 @@ contract L1BlockInterop is L1Block {
     ///         depositor account.
     /// @param _type  The type of configuration to set.
     /// @param _value The encoded value with which to set the configuration.
-    function setConfig(ConfigType _type, bytes calldata _value) external {
+    function setConfig(ConfigType _type, bytes calldata _value) public override {
         if (msg.sender != DEPOSITOR_ACCOUNT()) revert NotDepositor();
 
-        if (_type == ConfigType.SET_GAS_PAYING_TOKEN) {
-            _setGasPayingToken(_value);
-        } else if (_type == ConfigType.ADD_DEPENDENCY) {
+        if (_type == ConfigType.ADD_DEPENDENCY) {
             _addDependency(_value);
         } else if (_type == ConfigType.REMOVE_DEPENDENCY) {
             _removeDependency(_value);
+        } else {
+            L1Block.setConfig(_type, _value);
```

Comment:
We can directly use `super.setConfig(_type, _value);` at the beggining and avoid checking the `msg.sender` again